distribution/docker/build.gradle

```diff
@@ -396,7 +396,7 @@ void addBuildDockerImageTask(Architecture architecture, DockerBase base) {
 
       dockerContext.fileProvider(transformTask.map { Sync task -> task.getDestinationDir() })
 
-      noCache = BuildParams.isCi
+      noCache = BuildParams.isCi()
```

Comment:
Did this work before? I guess the correct property accessor should actually just be `BuildParams.ci`.